### PR TITLE
new: add 28093 iranian websites

### PR DESCRIPTION
**Pull request by hiddify1**

### CODE REVIEW

**Comment by rootmelo92118** [comment]
Could you give us the source?

**Comment by hiddify1** [comment]
In the first line i have mentioned that those files are based on 
# data from https://github.com/bootmortis/iran-hosted-domains/

https://github.com/v2fly/domain-list-community/blob/798ef1ec9a8738cf2597e4bdeb9dbde8a8575ae7/data/category-ir-other#LL2C1-L3C1

**Comment by bootmortis** [comment]
As I said before in other PRs, our domains in [bootmortis/iran-hosted-domains](https://github.com/bootmortis/iran-hosted-domains/) are updated weekly; If we are going to add those domains here, then this repo should be updated weekly to be synced with ours and reflect bug fixes; also, one of our sources [is already v2fly](https://github.com/bootmortis/iran-hosted-domains/blob/main/README.md#sources--acknowledgements), so I'm against from merging these two repositories.

**Comment by hiddify1** [comment]
no worry we can update it everytime
but see in the clients they put geosite.dat so it is not easy to ask old people with no knowledge to add it.

I suggest having a single source. similar to geosite-cn

**Comment by rootmelo92118** [comment]
@hiddify1 @bootmortis I have a question. It is from https://eservices.ito.gov.ir/page/iplist ?

**Comment by bootmortis** [comment]
> @hiddify1 @bootmortis I have a question. It is from https://eservices.ito.gov.ir/page/iplist ?

Yes, ITO GOV is one of our sources; you can see our sources full list [here](https://github.com/bootmortis/iran-hosted-domains/tree/main#sources--acknowledgements); I've mirrored ITO GOV [in here](https://github.com/bootmortis/ito-gov-mirror) and update it weekly. But we also have a filter list to remove some false domains ([like these](https://github.com/v2fly/domain-list-community/pull/1438#issuecomment-1428484966)) from our main repo.

**Comment by bootmortis** [comment]
> no worry we can update it everytime but see in the clients they put geosite.dat so it is not easy to ask old people with no knowledge to add another dat file to it.

I don't think the v2fly repo support automated update for their list.

**Comment by rootmelo92118** [comment]
I suggest both of you to see this conclusion.
https://github.com/v2fly/domain-list-community/pull/1600

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ on:
       - master
     paths-ignore:
       - "**/README.md"
+permissions:
+   contents: write
```
Comment (rootmelo92118) [inline .github/workflows/build.yml:9]: What are you doing?

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (hiddify1) [reply .github/workflows/build.yml:9]: i see that you do not want to merge this so i tried to build my own .dat file it needs the permission to release. I forgot that i didn't create a new branch. if you want to merge i will revert it :D

Comment (hiddify1) [reply .github/workflows/build.yml]: i have remove those changes

Comment (rootmelo92118) [reply .github/workflows/build.yml]: You can see README.md.
"""
A script will verify your pull request to test whether your PR is correct or not every time you update the PR. Only the PR which passes the test will be merged. Please go to the Action label to get detailed information if you didn't pass it. We also provide the file which has been generated to make you test.
"""
So you can get the file in GitHub action.

**Pull request closed without merging**